<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPaintCalibrationSaveParameter</class>
 <widget class="QDialog" name="DialogPaintCalibrationSaveParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save parameter</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelPressure">
      <property name="text">
       <string>Pressure:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditPressure"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelTemperature">
      <property name="text">
       <string>Temperature:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditTempressure"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTableView" name="tableViewPaintCalirbrationParameter"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>82</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPaintCalibrationSaveParameter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPaintCalibrationSaveParameter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
